assets/blocks/course-outline/status-preview/status-store.js

```diff
@@ -2,12 +2,13 @@
  * WordPress dependencies
  */
 import { select as selectData, registerStore } from '@wordpress/data';
+import { createReducerFromActionMap } from '../../../shared/data/store-helpers';
```

Comment:
Although I don't like the destinction between Wordpress and Internal dependencies, if it exists we should honor it 🙂